ext/export/src/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+option(BUILD_JA2EXPORT "Build the Ja2Export tool." ON)
+
+if(BUILD_JA2EXPORT)
```

Comment:
(I dont know best practices)
wouldnt it make more sense to skip the inclusion in the root CMakeLists instead of here?
would also make all what-to-build decisions be in a central location

Reply:
I like having the exporter tool decide whether to build itself or not. but there is a case for moving the `option(...)` to the root CMakeLists file.